### PR TITLE
[test] Add rom_e2e_boot_policy_bad_manifest

### PR DESCRIPTION
**Pull request by alphan**
Signed-off-by: Alphan Ulusoy <alphan@google.com>

### CODE REVIEW

File: sw/device/silicon_creator/rom/e2e/BUILD [hunk 1]
```diff
@@ -24,26 +24,34 @@ load(
...
+
+MSG_BOOT_POLICY_ROLLBACK = _BFV_TEMPLATE.format("0342500d")
 
-SLOT_A_OFFSET = "0x0"
+MSG_MANIFEST_BAD_ENTRY_POINT = _BFV_TEMPLATE.format("014d410d")
 
-SLOT_B_OFFSET = "0x80000"
+MSG_MANIFEST_BAD_CODE_REGION = _BFV_TEMPLATE.format("024d410d")
+
+SLOTS = {
```
Comment (timothytrippel) [inline sw/device/silicon_creator/rom/e2e/BUILD:51]: should this go in `rules/opentitan.bzl` ?

**Review by timothytrippel — APPROVED** [review]

File: sw/device/silicon_creator/rom/e2e/BUILD (hunk 1 above)
Comment (alphan) [reply sw/device/silicon_creator/rom/e2e/BUILD:51]: Probably but I just wasn't sure the BFV constants versus the repeating template. I should probably define the constants in `opentitan.bzl` and use them here to define repeated versions. Will do this in a separate PR if you don't mind.

**Comment by alphan** [comment]
Rebased (trivial), will merge once quick lint is complete:

image

File: sw/device/silicon_creator/rom/e2e/BUILD (hunk 1 above)
Comment (timothytrippel) [reply sw/device/silicon_creator/rom/e2e/BUILD:51]: sgtm!

**Pull request merged**